tests/UI/campaigns/functional/BO/02_orders/01_orders/viewAndEditOrder/07_messagesBlock.js

```diff
@@ -0,0 +1,399 @@
+require('module-alias/register');
+
+const {expect} = require('chai');
+
+// Import utils
+const helper = require('@utils/helpers');
+const testContext = require('@utils/testContext');
+
+// Import common tests
+const loginCommon = require('@commonTests/loginBO');
+const {createOrderByCustomerTest} = require('@commonTests/FO/createOrder');
+
+// Import BO pages
+const dashboardPage = require('@pages/BO/dashboard');
+const ordersPage = require('@pages/BO/orders');
+const viewOrderPage = require('@pages/BO/orders/view');
+const orderMessagesPage = require('@pages/BO/customerService/orderMessages');
+
+// Import FO pages
+const foLoginPage = require('@pages/FO/login');
+const foHomePage = require('@pages/FO/home');
+const foOrderConfirmationPage = require('@pages/FO/checkout/orderConfirmation');
+const foMyAccountPage = require('@pages/FO/myAccount');
+const foOrderHistoryPage = require('@pages/FO/myAccount/orderHistory');
+
+// Import demo data
+const {DefaultCustomer} = require('@data/demo/customer');
+const {PaymentMethods} = require('@data/demo/paymentMethods');
+
+const baseContext = 'functional_BO_orders_orders_viewAndEditOrder_messagesBlock';
+
+let browserContext;
+let page;
+
+const messageData = {orderMessage: 'Delay', displayToCustomer: true, message: ''};
+const secondMessageData = {orderMessage: 'Delay', displayToCustomer: false, message: 'test message visibility'};
+let textMessage = '';
+
+// Get today date format 'mm/dd/yyyy'
+const today = new Date();
+const mm = (`0${today.getMonth() + 1}`).slice(-2); // Current month
+const dd = (`0${today.getDate()}`).slice(-2); // Current day
+const yyyy = today.getFullYear(); // Current year
+const todayDate = `${mm}/${dd}/${yyyy}`;
```

Comment:
Maybe it's time to create a generic function instead of copy paste this code in multiple files?